<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventAdminMode</class>
 <widget class="QMainWindow" name="EventAdminMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EventPlanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please Create Your Event:</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="saveButton">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Event Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eventNameTextBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#qt_calendar_monthbutton,#qt_calendar_yearbutton{
color:
rgb(0, 0, 0);
}
QCalendarWidget QWidget#qt_calendar_navigationbar{
background-color:
rgb(255,255,255);
}
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="startTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="endTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End Time:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set24Hour">
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>24 Hour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set12Hour">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>12 Hour</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Format:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="timeSlotsWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Slots Added:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTimeSlots">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Time Slots</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearTimeSlotsButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Clear Time Slots</string>
    </property>
    <property name="minimumDate" stdset="0">
     <date>
      <year>2017</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="gridVisible" stdset="0">
     <bool>true</bool>
    </property>
    <property name="navigationBarVisible" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
